modules/utxo-lib/src/bitgo/nonStandardHalfSigned.ts

```diff
@@ -37,9 +29,12 @@ export function padInputScript(input: Input, signatureIndex: number): void {
   }
 
   const [op0, signatureBuffer, sigScript] = decompiledSigScript;
-
   if (op0 !== opcodes.OP_0 && !(Buffer.isBuffer(op0) && op0.length === 0)) {
-    throw new Error(`unexpected instruction`);
+    return;
```

Comment:
It's unclear to me when this method is supposed to return or throw an exception. Maybe you could add comments here on in the method comment block with the expected behavior.

Reply:
I changed my mind here and decided that it should not throw an exception when it hits a weird input that it doesn't recognize.